### DISCUSSION TITLE
Airflow operator

### DISCUSSION THREAD

**Discussion by kangshung** [kedro-org/kedro#1716] [Idea]
Hey, are there any plans to add KedroOperator to [an official Airflow provider list](https://github.com/apache/airflow/tree/main/airflow/providers)? This would really make this operator ["official"](https://airflow.apache.org/docs/). KedroSession, if needed, could be probably passed in a different way.

**Comment by datajoely** [comment]
There are now! I've just created issue #1717  which we will discuss at the next backlog grooming session.
